sematic/ui/packages/common/src/typeViz/views/hugging_face_reference.tsx

```diff
@@ -0,0 +1,106 @@
+import { OpenInNew } from "@mui/icons-material";
+import { Button, Tooltip } from "@mui/material";
+import { useTextSelection } from "@sematic/common/src/hooks/textSelectionHooks";
+import { ViewComponentProps} from "src/typeViz/common";
+
+type Reference = {
+    owner: string | null;
+    repo: string;
+    commit_sha: string | null;
+    subset?: string | null | undefined;
+}
+
+export function HuggingFaceModelReferenceShortView(props: ViewComponentProps) {
```

Comment:
Maybe you wanted this 

```
props: ValueComponentProps
```

instead of 
```
props: ViewComponentProps
```

ditto for line 24

Reply:
fixed